public/docs/_examples/quickstart/ts/app/app.module.ts

```diff
@@ -1,13 +1,16 @@
 // #docregion
-import { NgModule }      from '@angular/core';
-import { BrowserModule } from '@angular/platform-browser';
+import { Component, NgModule } from '@angular/core';
+import { BrowserModule }       from '@angular/platform-browser';
 
-import { AppComponent }   from './app.component';
+@Component({
+  template: `<h1>Hello Angular!</h1>`,
+  selector: 'my-app'
+})
+export class AppComponent { }
```

Comment:
I went back and forth on this. I decided to keep `export` even though not needed rather than have this be a difference later or make you wonder why `AppModule` is exported and `AppComponent` is not.